www/adblock2privoxy/Portfile

```diff
@@ -104,6 +137,29 @@ post-activate {
 	</array>\\
 &|" \
         ${prefix}/etc/${startupitem.location}/org.macports.${name}/org.macports.${name}.plist
+
+    foreach f [list \
```

Comment:
In previous cases where we've tried to achieve the same thing in MacPorts, we install the config files with a `.default` suffix and add a `post-activate` block that copies them to the unsuffixed version when they do not exist there yet.

See for example the squid4, torsocks, arb, molly-guard or varnish Portfiles that do that.

I'll leave it up to you whether you want to use the same approach, or this variant, just let me know.

Reply:
Thanks. I'd like to merge as is.

I'm very familiar with the `squid4`, `privoxy` and other Portfiles, and this one is very related and intended to work hand-in-hand with them, so I did think about using the `squid4` and `privoxy` approaches. 

Rather, I took this alternate approach because it accomplished several objectives:
* It doesn't confuse the issue about whether these example configurations are part of MacPorts or the upstream package. E.g. there actually [exists](https://wiki.squid-cache.org/SquidFaq/ConfiguringSquid) a `squid.conf.default`. But these files are highly specific to this port.
* It provides a way to (safely) initialize the configuration if there's a pre-existing one.